scipy/spatial/transform/tests/test_rotation.py

```diff
@@ -1635,18 +1631,23 @@ def test_slerp_rot_is_rotation():
         t = np.array([0, 1])
         Slerp(t, r)
 
+SLERP_MESSAGE = "must be a sequence of at least 2 rotations"
```

Comment:
Perhaps `SLERP_EXCEPTION_MESSAGE`.

Reply:
Renamed in c8a64228d69d13cb11315a00e7db32dbd0de686e